barretenberg/cpp/src/barretenberg/stdlib_circuit_builders/ultra_circuit_builder.cpp

```diff
@@ -41,6 +42,9 @@ template <typename Arithmetization> void UltraCircuitBuilder_<Arithmetization>::
      * our circuit is finalized, and we must not to execute these functions again.
      */
     if (!circuit_finalized) {
+        if (ensure_nonzero) {
+            add_gates_to_ensure_all_polys_are_non_zero();
```

Comment:
confusing that this wasn't renamed to add_ultra_gates_to_ensure_all_polys_are_non_zero

Reply:
It's a member function of UltraCircuitBuilder so it feels redundant.